### PR TITLE
devprod: add /api/wipe-db endpoint

### PR DESCRIPTION
**Pull request by jgehrcke**
This is motivated by https://github.com/conbench/conbench/issues/531. During local development it may be convenient to have an API endpoint that one can hit with the browser and/or curl. Tested this locally. Also added a test case.

README update follows.

CC @sayantikabanik

### CODE REVIEW

**Comment by jgehrcke** [comment]
Note that `empty_db_tables()` is tested implicitly between all tests, i.e. I only built a pragmatic test that confirms existence of the endpoint, not a test that confirms that this indeed wipes DB state.

**Comment by jgehrcke** [comment]
This is part of a series of smaller devprod PRs. Merging.

File: conbench/api/index.py [hunk 1]
```diff
@@ -9,11 +10,21 @@
...
 @api.route("/docs.json")
 def docs():
+    d = spec.to_dict()
+
+    # In TESTING mode there is a special endpoint that we do not want to mess
+    # with test_docs.
+    if "/api/wipe-db" in d["paths"]:
+        del d["paths"]["/api/wipe-db"]
+
     return f.jsonify(spec.to_dict())
```
Comment (austin3dickey) [inline conbench/api/index.py:28]: You wanted to `return f.jsonify(d)`, no?

Comment (jgehrcke) [reply conbench/api/index.py:28]: Yes. Thank you for your pair of eyes!!

Question is: why did the tests pass nevertheless? :D Looking now.

**Pull request merged**